modules/plugin/jdbc/jdbc-postgis/src/main/java/org/geotools/data/postgis/PostgisNGDataStoreFactory.java

```diff
@@ -156,6 +162,11 @@ protected JDBCDataStore createDataStoreInternal(JDBCDataStore dataStore, Map par
         // check geometry simplification (on by default)
         Boolean simplify = (Boolean) SIMPLIFY.lookUp(params);
         dialect.setSimplifyEnabled(simplify == null || simplify);
+        
+        // encode BBOX filter with wrapping ST_Envelope (GEOT-5167)
+        Boolean encodeBBOXAsEnvelope = (Boolean) ENCODE_BBOX_FILTER_USING_ENVELOPE.lookUp(params);
```

Comment:
Add note of parameter [PostGIS DataStore](http://docs.geotools.org/latest/userguide/library/jdbc/postgis.html) doc page.